application/src/main/java/net/furizon/backend/infrastructure/image/action/JooqAddMediaAction.java

```diff
@@ -0,0 +1,28 @@
+package net.furizon.backend.infrastructure.image.action;
+
+import lombok.RequiredArgsConstructor;
+import net.furizon.jooq.infrastructure.command.SqlCommand;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.util.postgres.PostgresDSL;
+import org.springframework.stereotype.Component;
+
+import static net.furizon.jooq.generated.Tables.MEDIA;
+
+@Component
+@RequiredArgsConstructor
+public class JooqAddMediaAction implements AddMediaAction {
+    private final SqlCommand sqlCommand;
+
+    @Override
+    public long invoke(@NotNull String mediaPath, @NotNull String mediaType) {
+        // TODO -> Better add media owner as well (userId)
```

Comment:
Yes, we could also add an uploaded timestamptz column, so that we can add a cleanup cronjob that cleans up unused uploads